src/api/Annotations.js

```diff
@@ -64,7 +65,7 @@ export default class Annotations extends MarkerBasedApi {
                 id: fileVersionId,
                 type: 'file_version',
             },
-            status: 'open',
+            status: ACTIVITY_FEED_ITEM_OPEN,
```

Comment:
Does the API allow for creating annotations with a status other than `open`? I'm wondering if this could be omitted

Reply:
It turns out that API doesn't allow custom status when creating Annotation, so `status` is not needed. And as per [docs](https://git.dev.box.net/AppServices/end-user-web/blob/master/schemas/v2/m0/models/annotations/createAnnotationPayload.json) `type` is also not needed. I've removed both.